artemis-server/src/test/java/org/apache/activemq/artemis/tests/util/SingleServerTestBase.java

```diff
@@ -42,6 +42,7 @@ public void setUp() throws Exception {
       super.setUp();
 
       server = createServer();
+      server.getConfiguration().setAddressQueueScanPeriod(100);
```

Comment:
I tried to do that, however there's quite a few...

I measure the time on the testsuite and this not cause any side effects on the time o other tests.

Reply:
Its not all about the time, but rather more that essentially _every_ test [not testing this] is now running with a non-represeatative broker config, which is something I think should generally be avoided - and is particularly shown up by the other tests that then had to artificially increase it again because it was set unexpectedly low everywhere by changes like this.

If the default is 30sec then only tests verifying this stuff should need to be setting the config (and then, typically lower than this 100ms).

Reply:
I removed this already. I'm only using it at the tests where it's needed.

Reply:
The diff still seems like it it setting it in a lot more places than I'd expect. This specific class, SingleServerTestBase, certainly seems like its intended to be more general than fitting this config being set in it.

Reply:
I thought I had cleaned it up on Saturday Night...

I will double make sure I will cleanup this using some quality time this time.. not while I was watching TV :) 

 .. thanks man